benchmarks/BenchmarkServer/README.md

```diff
@@ -0,0 +1,15 @@
+## Purpose
+
+This project is to assist in Benchmarking SignalR.
+It makes it easier to test local changes than having the App in the Benchmarks repo by letting us make changes in signalr branches and using the example commandline below to run the benchmarks against our branches.
```

Comment:
Add a link to the client code that the benchmark server works with, since we can't move that in here as well right now.